### PR TITLE
Tokens ERC20 connector support

### PR DESCRIPTION
**Pull request by calbritt**
- Similar implementation to [Tokens ERC1155 Connector Support PR](https://github.com/hyperledger/firefly-helm-charts/pull/4) and also does not have  IT test for ERC1155 currently. 

- added in github PAT note to initial README page

### CODE REVIEW

File: charts/firefly/templates/_helpers.tpl
Comment (hfuss) [inline charts/firefly/templates/_helpers.tpl:267]: Everything in this PR looks great! Only thing that might need correct is this bit. Since `tokens` is already getting configured for `erc1155` we'll want to just do a sort of `or` to append `erc20` config to the `tokens` plugin list if its set. So lines 255 - 272 should be replaced with something like:
```yaml
# realized we had a "bug" here, a user should be able to use `config.tokensOverride` even if either `erc1155` or `erc20` is enabled
{{- if and .Values.config.tokensOverride }}
tokens:
    {{- tpl .Values.config.tokensOverride . | nindent 2 }}
{{- else if or .Values.erc1155.enabled .Values.erc20.enabled }}
tokens:
  {{- if .Values.erc1155.enabled }}
  - plugin: fftokens
    name: erc1155
    url: http://{{ include "firefly.fullname" . }}-erc1155.{{ .Release.Namespace }}.svc:{{ .Values.erc1155.service.port }}
  {{- end }}
  {{- if .Values.erc20.enabled }}
    - plugin: fftokens
    name: erc20
    url: http://{{ include "firefly.fullname" . }}-erc20.{{ .Release.Namespace }}.svc:{{ .Values.erc20.service.port }}
  {{- end }}
{{- end }}
```

File: .github/workflows/helm.yml
Comment (hfuss) [inline .github/workflows/helm.yml:35]: We'll want to set both here so we test templating for both connectors while we wait for ethconnect support.

**Comment by hfuss** [comment]
@calbritt ah I see the DCO is failing, yeah this can be a bit annoying. You don't have to sign your commits via GPG or anything, you just have to do `git commit -s` on every commit to do a "sign off". I rebased and signed off on them for you based on https://duan.ca/2016/12/24/rebase-multiple-git-commits-with-signing/:
```
git rebase -i upstream/main
# put an edit at the beginning of every commit listed in the rebase

# repeat for all commits until the rebasing is done
git commit -s --amend --no-edit && git rebase --continue
```

**Comment by hfuss** [comment]
Thank you for the contribution @calbritt! Cleaned it up a bit so that we can start testing this. Will get a second review from @drewmarshburn before we get this merged.

**Review by hfuss — APPROVED** [review]

File: charts/firefly/templates/erc20/deployment.yaml
Comment (hfuss) [inline charts/firefly/templates/erc20/deployment.yaml:58]: We'll need to update ERC20 to include these endpoints like we did for ERC1155. Will get a PR open for that as I test this.

Comment (hfuss) [reply charts/firefly/templates/erc20/deployment.yaml]: Opened https://github.com/hyperledger/firefly-helm-charts/issues/11 instead as a follow up.

**Review by drewmarshburn — APPROVED** [review]
Good stuff @calbritt 🎉 . I'd say it was a successful first week!!

**Pull request merged**
